ivy/functional/frontends/torch/tensor_functions.py

```diff
@@ -14,8 +14,18 @@ def is_tensor(obj):
 # def is_storage(obj):
 # 	return ivy.is_storage(obj)
 
-# def is_complex(obj):
-# 	return ivy.is_complex(obj)
+
+def is_complex(input):
+    arr = ivy.asarray(input)
```

Comment:
The failures suggest that it could be because you are using `ivy.asarray` and not using the ivy.Array class init.